### ISSUE TITLE
Create view to allow Shelter owner to override info provided by admin on all shelters for which they are owners

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/ServiceNet#434]
Once #433 is complete, for a logged in Sacramento user:

* Add a item to the header menu called "My Shelter(s)". This should only be displayed for users who are owners of one or more shelter.

* When a user visits "my shelter(s)" page they should immediately see cards representing each of their associated shelters (similar to the homepage)

* Each of these cards should have an edit button in the lower right corner.

* When that user clicks the edit button, they should be brought to a page where they can update the existing data for that shelter (it can be similar to the default entity edit screen available to admins)

The page should have a "Cancel" and "Update" button which brings the user back to the My Shelter page

When a Sacramento user logs in to the platform, any shelters for which they are an "owner"

**Comment by johnhbenetech** [comment]
@awalkowiak can we attach commits to issues when possible

A few issues to check on this one:

From a security standpoint, it would be great if this was server side validated. Currently, anyone can append '/edit' to the end of any shelter ID and make changes (even if they are not an owner). Ideally a non-owner would not even be able to access the edit page let alone commit changes.


Secondly, this is currently exacerbated on the front end through the following bug:
* Add a sac user as an owner of a shelter
* Go to 'My Shelters' page and see the single owned shelter
* Click 'edit', make some changes, click save

- User is then brought back to the 'My Shelters' page, but it lists all shelters with edit links, and allows the user to update them for reasons above.

**Comment by awalkowiak** [comment]
@johnhbenetech The was no QA on this issue yet, but I'll fix listed points before it.

**Comment by bognasoldev89** [comment]
I confirm that all mentioned functionalities are developed. "My Shelters" page for shelters' owners is available and working

![434 1](https://user-images.githubusercontent.com/51324797/69057195-ba078c00-0a11-11ea-9561-78c40cc994e2.png)

It is not available for sacramento users without shelter's ownership.

I was able to edit shelters and changes were visible:

![434 2](https://user-images.githubusercontent.com/51324797/69057332-f33ffc00-0a11-11ea-9a4a-a4343cd492d1.gif)

I also checked for security issues that were mentioned and it looks like those no longer occur.

@johnhbenetech I noticed that you didn't finish sentence in ticket description: _When a Sacramento user logs in to the platform, any shelters for which they are an "owner"_ What did you mean?
